src/map.rs

```diff
@@ -301,13 +301,13 @@ fn extract_maps<L: PxLayer>(
     }
 }
 
-pub(crate) type TileComponents = (&'static PxTile, Option<&'static PxFilter>);
+pub(crate) type TileComponents = (&'static PxTile, Option<&'static PxFilter>, Option<&'static PxFlip>);
```

Comment:
Make this non-optional and add `PxFlip` as a required component on `PxTile`

Reply:
Can do.